src/server/AuthorizingHttpHandler.ts

```diff
@@ -77,11 +83,25 @@ export class AuthorizingHttpHandler extends OperationHttpHandler {
       await this.authorizer.handleSafe({ credentials, requestedModes, availablePermissions });
     } catch (error: unknown) {
       this.logger.verbose(`Authorization failed: ${(error as any).message}`);
+      if (error instanceof HttpError) {
```

Comment:
There is an `HttpError.isInstance` function. There are some potential issues with `instanceof` and components.js https://github.com/CommunitySolidServer/CommunitySolidServer/issues/464.

(Although I don't know how much this can actually occur in practice).

Reply:
```suggestion
      if (HttpError.isInstance(error)) {
```